Makefile

```diff
@@ -39,4 +39,4 @@ clean:
 documentation:
 	make project
 	@gem list jazzy -i > /dev/null || gem install jazzy || { echo "ERROR: Failed to locate or install the ruby gem jazzy; please install yourself with 'gem install jazzy' (you may need to use sudo)"; exit 1; }
-	jazzy --module MongoSwift --module-version 0.0.2 --root-url https://mongodb.github.io/mongo-swift-driver/ --documentation Development.md
+	jazzy --module MongoSwift --module-version 0.0.3 --root-url https://mongodb.github.io/mongo-swift-driver/ --documentation Development.md
```

Comment:
Since 0.0.3 is already tagged and we'll need to create a new tag following this change, shouldn't this be 0.0.4?